manim/scene/scene.py

```diff
@@ -113,9 +117,35 @@ def tear_down(self):
         pass
 
     def construct(self):
-        """
-        The primary method for constructing (i.e adding content to)
-        the Scene.
+        """Add content to the Scene.
+
+        From within :meth:`Scene.construct`, display mobjects on screen by calling
+        :meth:`Scene.add` and remove them from screen by calling :meth:`Scene.remove`.
+        All mobjects currently on screen are kept in :attr:`Scene.mobjects`.  Play
+        animations by calling :meth:`Scene.play`.
+
+        Notes
+        -----
+        Initialization code should go in :meth:`Scene.setup`.  Termination code should
+        go in :meth:`Scene.tear_down`.
```

Comment:
**Question:** Are you _sure_ about this? isn't setup stupposed to be used by subclasses of Scene?

Reply:
I believe `setup` is meant to be used by subclasses especifically for the purpose of containing initialization code (that may be shared among different subclasses). But even then, it makes sense for it to also contain the user's initialization code. If a user subclass overrides `setup`, then it's on the user to remember to call `super().setup()` if they need to.